index.html

```diff
@@ -196,6 +196,18 @@ <h1 class="used-by">Used By</h1>
 		</div>
 	</section>
 
+	<!-- Adopter Section -->
+	<section class="adopter-section">
+		<div class="adopter-container">
+            <h3 class="adopter-title">Add yourself as an adopter...</h3>
+            <h5 class="adopter-description">and share your service mesh story with the community</h5>
+			<p class="button adopter-button">
+				<a href="https://github.com/layer5io/meshery/blob/master/ADOPTERS.md" target="_blank">
+					Become a Adopter
+				</a>
+			</p>
```

Comment:
```suggestion
			<div class="button adopter-button">
				<a href="https://github.com/layer5io/meshery/blob/master/ADOPTERS.md" target="_blank">
					Become a Adopter
				</a>
			</div>
```

Also, fix Indentation.

Reply:
```suggestion
			<p class="button adopter-button">
				<a href="https://github.com/layer5io/meshery/blob/master/ADOPTERS.md" target="_blank">
					Become an Adopter
				</a>
			</p>
```

Reply:
P.S. Currently the section looks like this.

![screenshot-deploy-preview-369--mesheryio netlify app-2020 11 03-00_55_00](https://user-images.githubusercontent.com/31549564/97907353-5fc12180-1d6f-11eb-86e3-af86e10c853a.png)